pkg/virt-operator/install-strategy/create.go

```diff
@@ -110,10 +111,26 @@ func controllerDeployments(strategy *InstallStrategy) []*appsv1.Deployment {
 	return deployments
 }
 
+func isValidLabel(label string) bool {
+	// First and last character must be alphanumeric
+	// middle chars can be alphanumeric, or dot hyphen or dash
+	// entire string must not exceed 63 chars
+	r := regexp.MustCompile(`^([a-z0-9A-Z]([a-z0-9A-Z\-\_\.]{0,61}[a-z0-9A-Z])?)?$`)
```

Comment:
Nit: could be shortened with: `^([[:alnum:]]([\w\-\.]{0,61}[[:alnum:]])?)?$` (though `len("[a-z0-9A-Z]") == len("[[:alnum:]]")` and `\w` might be a bit obfuscating...)